hack/versions-management/tag_reconciler.sh

```
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -o nounset
+set -o errexit
+set -o pipefail
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+"$SCRIPT_DIR/github_auth.sh"
+
+python "$SCRIPT_DIR/tag_reconciler.py" $([ "$DRY_RUN" = true ] && echo "--dry-run")
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)